e2e/scaffolding/helpers.ts

```diff
@@ -52,6 +52,11 @@ export const CENTS = 1000000n;
 export const DOLLARS = 100n * CENTS;
 export const STARTING_BALANCE = 6n * CENTS + DOLLARS;
 
+export function getTokenPerCapacity(): bigint {
+  // Perbil
+  return 1_000_000_000n / ExtrinsicHelper.api.consts.capacity.capacityPerToken.toBigInt();
```

Comment:
My guess is that **capacityPerToken** should be `20_000_000`  to make it equal to the old value. Is that correct?

Reply:
Correct!

Reply:
2% Per billion = `20_000_000`
<img width="768" alt="image" src="https://github.com/user-attachments/assets/1a4e34cf-b32b-4ffd-9114-aea7680653dd">